Assets/LeapMotion/Core/Scripts/Attributes/QuickButtonAttribute.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Leap.Unity.Query;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Leap.Unity.Attributes {
+
+  public class QuickButtonAttribute : CombinablePropertyAttribute, IBeforeFieldAdditiveDrawer {
+
+    public const float PADDING_RIGHT = 12f;
+
+    public string label = "Quick Button";
+    public string methodOnPress = null;
+    public string tooltip = "";
+
+    public Type type;
```

Comment:
These have no need to be variables I think?  They are only used within the Draw method?

Reply:
le done